src/Plutip/Server.purs

```diff
@@ -162,8 +181,60 @@ startOgmios cfg params = liftEffect $ spawn "ogmios" ogmiosArgs
     , params.nodeConfigPath
     ]
 
-stopOgmios :: ChildProcess -> Aff Unit
-stopOgmios = liftEffect <<< kill SIGINT
+stopChildProcess :: ChildProcess -> Aff Unit
+stopChildProcess = liftEffect <<< kill SIGINT
+
+startPlutipServer :: PlutipConfig -> Aff ChildProcess
+startPlutipServer cfg = do
+  p <- liftEffect $ spawn "plutip-server" [ "-p", UInt.toString cfg.port ]
+    defaultSpawnOptions { detached = true }
+  delay $ Milliseconds 100.0
+  pure p
+
+startPostgresServer
+  :: PostgresConfig -> ClusterStartupParameters -> Aff ChildProcess
+startPostgresServer pgConfig _ = do
+  randomStr <- liftEffect $ fromJustEff "hello" =<< head <$> randomSample
+    genAlphaString
+  tmpDir <- liftEffect tmpdir
+  let
+    workingDir = tmpDir <> "/" <> randomStr
+    databaseDir = workingDir <> "/postgres/data"
+  liftEffect $ void $ execSync ("initdb " <> databaseDir) defaultExecSyncOptions
+  pgChildProcess <- liftEffect $ spawn "postgres"
+    [ "-D"
+    , databaseDir
+    , "-p"
+    , UInt.toString pgConfig.port
+    , "-h"
+    , pgConfig.host
+    , "-k"
+    , workingDir <> "/postgres"
+    ]
+    defaultSpawnOptions
+  delay (Milliseconds 3000.0)
```

Comment:
I'd like to use [ShareStream](https://pursuit.purescript.org/packages/purescript-node-child-process/9.0.0/docs/Node.ChildProcess#v:ShareStream) on the process stdout to catch the moment it starts listening on port, if that's possible. Please add a TODO in comments and create a new issue (the TODO should include a link to it).